mopidy/mpd/protocol/audio_output.py

```diff
@@ -43,7 +42,11 @@ def toggleoutput(context, outputid):
 
         Turns an output on or off, depending on the current state.
     """
-    pass
+    if outputid == 0:
```

Comment:
Cool that you are fixing this, but next time I would make it a separate commit within the PR as this is a distinct change from the main thing you are changing in the PR.